app/assets/javascripts/dashboard.js

```diff
@@ -54,6 +61,16 @@ MinionPoller = {
     });
   },
 
+  enable_kubeconfig: function(enable) {
+    if (enable) {
+      $("#download-kubeconfig").unbind('click')
+      $("#download-kubeconfig").removeAttr("disabled");
+    } else {
+      $("#download-kubeconfig").bind('click', function(e) { e.preventDefault(); });
+      $("#download-kubeconfig").attr("disabled", true);
+    }
+  },
```

Comment:
> Weird, it didn't work for me, that's why I did it :confused:

Just realized that it's an anchor using `.btn-link`. That's why it didn't work. You can just use `.attr('disabled', !enabled)` and jQuery will automatically remove if `false`.

> Even if disabled, I could still click it and then I got the download dialog... So it looks like it...

I would suggest a global event listener for all elements with `disabled` attribute like

```js
$('body').on('click', '[disabled]', function(e) {
  e.preventDefault();
});
```

It will make this method cleaner.